frontend/src/Resolver.tsx

```diff
@@ -1,11 +1,12 @@
-import "./App.css";
-import { useEffect, useMemo } from "react";
-import { useParams } from "react-router-dom";
-import { Loader } from "./components";
-import { hexToString } from 'useink/utils';
-import { useLinkContract } from "./hooks";
-import { AbiMessage, ContractExecResult, Registry } from "useink/core";
-import { useAbiMessage } from "useink";
+import './App.css';
+import { useEffect, useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from './components';
+
+import { useLinkContract } from './hooks';
+import { AbiMessage, ContractExecResult, Registry } from 'useink/core';
+import { useAbiMessage } from 'useink';
+import { hexToString } from './lib/converter';
```

Comment:
can you explain why you changed the `hexToString` import?

Reply:
it seems the new version of useink doesnt export that function